bazel/foreign_cc/BUILD

```diff
@@ -43,6 +43,7 @@ cmake_external(
         "ENABLE_STATIC_LIB": "on",
         "ENABLE_LIB_ONLY": "on",
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_INSTALL_LIBDIR": "lib",
```

Comment:
This is due to in some platform it will pick `lib64` and bazel won't pick built `.a` file. It was defined in previous `nghttp2.sh`